tests/properties/expectation_values_test.cpp

```diff
@@ -51,3 +59,63 @@ BOOST_AUTO_TEST_CASE ( two_electron_throw ) {
     BOOST_CHECK_THROW(GQCP::calculateExpectationValue(g, d_invalid), std::invalid_argument);
     BOOST_CHECK_NO_THROW(GQCP::calculateExpectationValue(g, d_valid));
 }
+
+BOOST_AUTO_TEST_CASE ( mulliken_N2_STO_3G ) {
+
+    // Check that the mulliken population of N2 is 14 (N)
+
+    // Initialize the molecule and molecular Hamiltonian parameters for N2
+    GQCP::Atom N_1 (7, 0.0, 0.0, 0.0);
+    GQCP::Atom N_2 (7, 0.0, 0.0, GQCP::units::angstrom_to_bohr(1.134));  // from CCCBDB, STO-3G geometry
+    std::vector<GQCP::Atom> atoms {N_1, N_2};
+    GQCP::Molecule N2 (atoms);
+
+    auto ao_basis = std::make_shared<GQCP::AOBasis>(N2, "STO-3G");
+    auto ham_par = GQCP::constructMolecularHamiltonianParameters(ao_basis);
+
+    size_t K = ao_basis->get_number_of_basis_functions();
+
+    // We include all basis functions
+    GQCP::Vectoru gto_list (K);
+    for(size_t i = 0; i<K; i++){
+        gto_list[i] = i;
+    }
+
+    GQCP::OneElectronOperator mulliken = ham_par.calculateMullikenOperator(gto_list);
+
+    size_t N = N2.get_N();
+
+    // Create a 1-RDM for N2
+    Eigen::MatrixXd D = Eigen::MatrixXd::Zero(K, K);
+    GQCP::OneRDM one_rdm = GQCP::calculateRHF1RDM(K, N);
+
+    double mulliken_population = GQCP::calculateExpectationValue(mulliken, one_rdm);
+    BOOST_CHECK(std::abs(mulliken_population - (N)) < 1.0e-08);
+
+    // Repeat this for the RDM of a DOCI expansion
+
+
+    // Solve the SCF equations
+    GQCP::PlainRHFSCFSolver plain_scf_solver (ham_par, N2);  // The DIIS SCF solver seems to find a wrong minimum, so use a plain solver instead
```

Comment:
```suggestion
    GQCP::PlainRHFSCFSolver plain_scf_solver (ham_par, N2);  // the DIIS SCF solver seems to find a wrong minimum, so use a plain solver instead
```

Could you also change this in my other test?